tests/Integration/Monitoring/TestMonitoringSystem.py

```diff
@@ -0,0 +1,139 @@
+"""
+It is used to test client->db-> service
+"""
```

Comment:
Give some more details, like what's necessary to be installed BEFORE running this integration test.